homeworks/Homework2/Date.cpp

```diff
@@ -0,0 +1,524 @@
+#include "Date.h"
+
+Date::Date()
+{
+	this->day = 1;
+	this->month = 1;
+	this->year = 1970;
+}
+
+Date::Date(int day, int month, int year)
+{
+	this->day = day;
+	this->month = month;
+	this->year = year;
+}
+
+int Date::Day()
+{
+    return this->day;
+}
+
+int Date::Month()
+{
+    return this->month;
+}
+
+int Date::Year()
+{
+    return this->year;
+}
+
+void Date::Day(int day)
+{
+    this->day = day;
+}
+
+void Date::Month(int month)
+{
+    this->month = month;
+}
+
+void Date::Year(int year)
+{
+    this->year = year;
+}
+
+void Date::addDay() {
+	switch (this->month)
+	{
+	case 1:
+		if (this->day == 31)
+		{
+			this->day = 1;
+			this->month++;
+		}
+		else {
+			this->day++;
+		}
+		break;
+	case 2:
+		if (this->day == 28)
+		{
+			this->day = 1;
+			this->month++;
+		}
+		else {
+			this->day++;
+		}
+		break;
+	case 3:
+		if (this->day == 31)
+		{
+			this->day = 1;
+			this->month++;
```

Comment:
multiple case 
switch(Answer)
{
    case 1: case 2: case 3: case 4:
        cout << "You need more cars.";
        break;
    ...
}